README.md

```diff
@@ -1,3 +1,10 @@
 # zulu
 [![Build Status](https://travis-ci.org/DE-labtory/zulu.svg?branch=master)](https://travis-ci.org/DE-labtory/zulu)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Language](https://img.shields.io/badge/language-go-orange.svg)](https://golang.org) [![Coverage Status](https://coveralls.io/repos/github/DE-labtory/zulu/badge.svg?branch=develop)](https://coveralls.io/github/DE-labtory/zulu?branch=develop)
\ No newline at end of file
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Language](https://img.shields.io/badge/language-go-orange.svg)](https://golang.org) [![Coverage Status](https://coveralls.io/repos/github/DE-labtory/zulu/badge.svg?branch=develop)](https://coveralls.io/github/DE-labtory/zulu?branch=develop)
+
+Zulu is a general unified crypto wallet api user can easily manage.
+
+* **Support various blockchain with unified interface**: Zulu basically support ethereum (as well as  its ERC20 token), bitcoin. Regardless of blockchain specific domain, Zulu provide simple and unified interface.
+* **Provide core wallet functionaility with minimal state**: Zulu as a third party library, tries to keep its state as few as possible still provide core crypto wallet functionality.
```

Comment:
I think that it is better to replace `third party library` with other word.

Reply:
@hihiboss Thanks for feedback